contrib/get_tools.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -euo pipefail
+
+KUBE_VERSION=v1.18.8
+
+echo ">>> Installing k3sup"
+sudo curl -sLS https://get.k3sup.dev | sh
+sudo install k3sup /usr/local/bin/
+
+echo ">>> Installing kubectl $KUBE_VERSION"
+OS=linux
+curl -sSLfo kubectl https://storage.googleapis.com/kubernetes-release/release/$KUBE_VERSION/bin/$OS/amd64/kubectl && \
+chmod +x kubectl && \
+sudo mv kubectl /usr/local/bin/
+
+echo ">>> Installing arkade"
```

Comment:
If we are installing arkade, why not download kubectl with arkade?

```
arkade get kubectl --version $KUBE_VERSION
mv $HOME/.arkade/bin/kubectl /usr/local/bin/
```

Reply:
done, thank you 🙏